backup.gemspec

```diff
@@ -36,13 +36,13 @@ Gem::Specification.new do |gem|
   gem.add_dependency "net-ssh", "3.2.0"
   gem.add_dependency "net-scp", "1.2.1"
   gem.add_dependency "net-sftp", "2.1.2"
-  gem.add_dependency "mail", "2.6.3" # patched
+  gem.add_dependency "mail", "2.6.4" # patched
```

Comment:
The upgrade to this means we do need to update the monkeypatch to be like this:
source: https://github.com/mikel/mail/compare/2.6.3...2.6.4#diff-70e36d3e2b37ee7aca19125cd388fd6fR47
`lib/mail/network/delivery_methods/exim.rb`

```ruby
# lib/backup/notifier/mail.rb:241
io.puts ::Mail::Utilities.to_lf(encoded_mail)
```